pegasus/src/env.rb

```diff
@@ -45,15 +48,19 @@ def self.create_logger
 def load_pegasus_settings
   $log = Pegasus.logger
 
-  I18n::Backend::Simple.send(:include, I18n::Backend::Fallbacks)
   if rack_env?(:development) && !CDO.load_locales
     I18n.load_path += Dir[cache_dir('i18n/en-US.yml')]
     I18n.load_path += Dir[cache_dir('i18n/es-ES.yml')]
   else
     I18n.load_path += Dir[cache_dir('i18n/*.yml')]
   end
   I18n.enforce_available_locales = false
-  I18n.backend.load_translations
+  I18n::Backend::Simple.send(:include, I18n::Backend::Fallbacks)
+  I18n::Backend::FileKeyValue.send(:include, I18n::Backend::Fallbacks)
```

Comment:
why are we still working on the Simple backend here, rather than just the new one?

Reply:
good point- probably cleaner to call #include on `I18n.backend.class`, so the fallback behavior is included regardless of the underlying backend implementation class.

Reply:
updated in 9e9eb31.